### PR TITLE
Feature gcp bucket source

### PR DESCRIPTION
**Pull request by solita-wihinenh**
Cluster plots and basic info file read from gcp buckets. When doing local development you can define "use_gcp_buckets = False" in config files. gevents had to be replaced with a different worker due to issues in bucket reading.

### CODE REVIEW

File: server/server.py [hunk 1]
```diff
@@ -94,16 +96,23 @@ def clusterplot(plot_type, variant):
         arr[0] = 'X' if arr[0] == '23' else arr[0] 
         exists_in_chip = fetch.check_var_in_chip(variant)
         filename = config['cluster_plots_location'] + '/' + plot_type + '/' + '_'.join(arr) + '.png'
-        with open(filename, 'rb') as f:
-            blob = f.read()
+        if (config['use_gcp_buckets']):
+            blob = cloud_storage.read_file(config['cluster_plots_bucket'], filename)
+            if blob is None:
+                raise FileNotFoundError("Requested cluster plot not found!")
```
Comment (juhis) [inline server/server.py:102]: should we abort here instead of raise like on the lines 109/112/114

**Review by juhis — CHANGES_REQUESTED** [review]
Great! One comment

File: server/server.py (hunk 1 above)
Comment (solita-wihinenh) [reply server/server.py:102]: I wanted to use the same error message as on 112/114. This raises an exception which is then caught on line 110.

Comment (juhis) [reply server/server.py:102]: True, was blind. Great!

**Review by juhis — APPROVED** [review]
Solid

**Pull request merged**